libbeat/beat/events/util.go

```diff
@@ -0,0 +1,50 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package events
+
+import "github.com/elastic/beats/v7/libbeat/beat"
+
+const (
+	FieldMetaID       = "_id"
+	FieldMetaIndex    = "index"
+	FieldMetaRawIndex = "raw_index"
+	FieldMetaAlias    = "alias"
+	FieldMetaPipeline = "pipeline"
```

Comment:
docstrings:

- `_id`: set event id. If `_id` is set we use the X op in the bulk request.
- `alias`, `index`: ... no idea :)
- `raw_index`: set the index name for the input
- `pipeline`: set the ingest node pipeline name to use for this event.